homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -77,6 +79,9 @@ class SensorType:
     "pressure": SensorType(
         unit=PRESSURE_HPA, icon=None, device_class=DEVICE_CLASS_PRESSURE
     ),
+    "load_power": SensorType(
```

Comment:
Is this change related to the new switches?

Reply:
yes, the switches have the load_power attribute